### ISSUE TITLE
Latest Version of EA Post Grid not Working

### ISSUE THREAD

**Issue by RozanneW** [rupok/essential-addons-for-elementor-lite#413]
Hi, so I've noticed that the EA post grid element has been glitchy for a while now. At this stage, the hover card isn't working when I hover over it, the image that goes with the grid block doesn't even link to the post when hovering over it.

After reverting to backups on my site, I suspect the problem is due to an incompatibility with Elementor's latest versions. I've noticed that the glitches become worse when updating to latest versions of Elementor and EA -- unfortunately I don't have backups that go back far enough to fix the issue.

If I try to fix the problem by adjusting things in Elementor, the post grid often turns into a messy list with part of the content hiding under my header section, so I've given up on fixing the issue.  I wish I hadn't updated my plugins,. The EA post grid is just breaking more with each update and I don't have sufficient coding knowledge to find the source of these issues.

**Comment by GitMubarak** [comment]
Can you please upgrade to latest version and check again? Let me know if upgrading resolves your issue or not.

**Comment by RozanneW** [comment]
Hi GitMubarak. Thank you for your reply. Much appreciated.

At this stage I'm quite scared of touching and updating anything. I've made quite a few changes to my website since reporting this issue and would like to keep these updates intact. I'll create a backup and get back to you on this issue.

How will we go about trying to fix the problem by upgrading? I have actually tried this and it makes things worse, so I'm hesitant to try this again. In the meantime, I've been using a different plugin to display my blog posts in grid style. But I use EA elements in other areas of my site, so it would be nice if I could go back to using it to display my posts, as I wouldn't need a separate plugin for it then.
